tools/build_pgo.sh

```diff
@@ -34,4 +34,8 @@ python tools/pgo_scripts/test_utility_scale.py
 
 deactivate
 
-${HOME}/.rustup/toolchains/*$arch*/lib/rustlib/$arch*/bin/llvm-profdata merge -o "$out_path" "$work_dir"
+if [[ `uname -m` == "aarch64" ]]; then
+    ${HOME}/.rustup/toolchains/1.79-aarch64-unknown-linux-gnu/lib/rustlib/$arch*/bin/llvm-profdata merge -o "$out_path" "$work_dir"
+else
+    ${HOME}/.rustup/toolchains/*$arch*/lib/rustlib/$arch*/bin/llvm-profdata merge -o "$out_path" "$work_dir"
+fi
```

Comment:
I'm not a big fan of how we're hard-coding the paths to the Rust toolchain in this file, making assumptions that we used MSRV for AArch64 - feels very easy to get out-of-sync and harder to update in the future.  Can we refactor slightly so that the `install_rust_{,_msrv}.sh` or relevant GitHub Actions jobs detect the installation location of `llvm-profdata` at install time, and feed the path into this script by command-line argument?

Reply:
I fixed this in: https://github.com/Qiskit/qiskit/pull/13737/commits/fae956b52c0198c73d587a816d2101635c945b9e I just made sure in the msrv path we only install a single toolchain so the glob find the correct path.